### PR TITLE
ICMSLST-506 Add emails for constabulary

### PR DESCRIPTION
**Pull request by kevinetienne**
Create/edit/delete constabulary emails and record responses with attachments. Constabulary emails are used to check user certificates and verified certificates.

### CODE REVIEW

File: config/settings/non_prod_base.py [hunk 1]
```diff
@@ -22,3 +22,4 @@
 # used in email content
 ILB_CONTACT_EMAIL = env.str("ICMS_ILB_CONTACT_EMAIL", "enquiries.ilb@icms.trade.dev.uktrade.io")
 ILB_CONTACT_PHONE = env.str("ICMS_ILB_CONTACT_PHONE", "N/A")
+FIREARMS_HOMEOFFICE_EMAIL = env.str("ICMS_FIREARMS_HOMEOFFICE", "firearms-homeoffice@example.com")
```
Comment (oskusalerma) [inline config/settings/non_prod_base.py:25]: the variable should be named the same both sides. `ICMS_FIREARMS_HOMEOFFICE_EMAIL` i guess?

File: config/settings/production.py [hunk 2]
```diff
@@ -32,6 +32,7 @@
 EMAIL_FROM = env.str("ICMS_EMAIL_FROM")
 ILB_CONTACT_EMAIL = env.str("ICMS_ILB_CONTACT_EMAIL")
 ILB_CONTACT_PHONE = env.str("ICMS_ILB_CONTACT_PHONE")
+FIREARMS_HOMEOFFICE_EMAIL = env.str("ICMS_FIREARMS_HOMEOFFICE")
```
Comment (oskusalerma) [inline config/settings/production.py:35]: same

File: web/domains/case/_import/firearms/forms.py [hunk 3]
```diff
@@ -166,3 +171,56 @@ def __init__(self, *args, **kwargs):
...
+        )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        choices = [
+            (c.email, f"{c.name} ({c.email})") for c in Constabulary.objects.filter(is_active=True)
+        ]
+        self.fields["email_to"].choices = choices
+
+        initial = [("firearms@homeoffice.gsi.gov.uk", "firearms@homeoffice.gsi.gov.uk")]
```
Comment (oskusalerma) [inline web/domains/case/_import/firearms/forms.py:208]: should this not be in the settings file?

**Review by oskusalerma — CHANGES_REQUESTED** [review]

File: config/settings/non_prod_base.py (hunk 1 above)
Comment (kevinetienne) [reply config/settings/non_prod_base.py:25]: double inconsistency!

File: web/domains/case/_import/firearms/forms.py (hunk 3 above)
Comment (kevinetienne) [reply web/domains/case/_import/firearms/forms.py]: Thanks fixed in https://github.com/uktrade/icms/pull/317/commits/74c65c783be031e98c2fd484c1320c0d14d15d27

**Pull request merged**